packages/run-protocol/test/vaultFactory/test-vaultFactory.js

```diff
@@ -965,8 +965,11 @@ test('interest on multiple vaults', async t => {
   await manualTimer.tickN(8);
   await eventLoopIteration();
 
+  console.log('DEBUG getUpdateSince() on assetNotifier');
```

Comment:
@gibson042 the following line hangs just as in https://github.com/Agoric/agoric-sdk/pull/5695/files#r911476188